packages/specs/schema-async-api/readme.md

```
@@ -0,0 +1,78 @@
+  <a href="https://tsed.dev/">Website</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://tsed.dev/getting-started/">Getting started</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://slack.tsed.dev">Slack</a>
+  <span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+  <a href="https://twitter.com/TsED_io">Twitter</a>
+</div>
+
+<hr />
+
+A package of Ts.ED framework. See website: https://tsed.dev/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)